widget/tabcontainer_test.go

```diff
@@ -1,33 +1,37 @@
-package widget
+package widget_test
 
 import (
 	"testing"
 
+	"fyne.io/fyne/internal/driver"
```

Comment:
This imports are not in order

Reply:
the VSCode Go integration groups imports by internal (Go), external and local. That's why it went there.

Reply:
Apologies, if goimports does this then I should have highlighted the row beneath - "widget" is not an internal package.